.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+## Contribute to Magento Glossary
+
+Share your knowledge with the community by contributing to Magento Glossary, part of Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [Glossary](https://github.com/magento/glossary) GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new terms.
+
+Magento's tech writer team and [Community Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/community-maintainers) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards for contributions
```

Comment:
Let's confirm this first. I don't see the glossary repo in the list of availablie projects in the EngComm dashboard that tracks contributions.

Reply:
@lorikrell can you help with verifying this?

Reply:
@jcalcaben, I contacted the EngComm tea about this via Slack.